robottelo/helpers.py

```diff
@@ -253,3 +256,84 @@ def remove_katello_ca(hostname=None):
             )
             if result.return_code != 0:
                 raise AssertionError('Failed to reset the rhsm.conf')
+
+
+def get_available_capsule_port(port_pool=None):
+    """returns a list of unused ports dedicated for fake capsules
+    This calls a fuser command on the server prompting for a port range. fuser
+    commands returns a list of ports which have a PID assigned (a list of ports
+    which are already used). This function then substracts unavailable ports
+    from the other ones and returns one of available ones randomly.
+
+    :param port_pool: A list of ports used for fake capsules (for RHEL7+: don't
+        forget to set a correct selinux context before otherwise you'll get
+        Connection Refused error)
+
+    :return: Random available port from interval <9091, 9190>.
+    :rtype: int
+    """
+    if port_pool is None:
+        port_pool = eval(settings.fake_capsules.port_range)
```

Comment:
Using eval is not good. The settings can be just `<start_port>,<end_port>` And the settings module can cast it to the expected format.